Qt.py

```diff
@@ -893,9 +893,14 @@ def load(self, uifile, *args, **kwargs):
                 etree = ElementTree()
                 etree.parse(uifile)
 
-                return Qt._QtUiTools.QUiLoader.load(
+                widget = Qt._QtUiTools.QUiLoader.load(
                     self, uifile, *args, **kwargs)
 
+                # Prevent RuntimeError in Nuke 9.0v9 with PySide 1.0.9
+                Qt.QtCompat._loadUi_temp = widget.parentWidget()
```

Comment:
Ok, in that case something like this should do the trick.

```python
                # Workaround for PySide 1.0.9, see issue #209
                Qt.QtCompat._loadUi_temp = widget.parentWidget()
```

Are we sure we need to store the variable, could we not just call `parentWidget()`? If not, could we store it in a local variable, e.g. `_workaround = ...`

What I'm looking for is our future selves not reading this and wonder what's what.